index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="background-img">
+
+    <header>
+      <div class="header-flex">
+        <div class="logo-link"><a class="logo-link" href="https://github.com/Khawanin">Khawanin</a></div>
+        <div class="union-icon"><img src="./Images/Union.png" alt="union icon"></div>
+      </div>
+    </header>
+
+    <main>
+      <article>
+        <h1 class="heading">I'm Khawanin <br>Glad to see <br>you!</h1>
+        <p class="self-intro">I’m a software developer! I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact me.</p>
+      </article>
+      <section>
+        <h2 class="connect">Let's connect</h2>
+        <ul class="social-icons">
+          <li><a href="https://twitter.com/home"><img src="./Images/twitter.png" alt="twitter icon"></a></li>
+          <li><a href="https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/"><img src="./Images/linkedin.png"
+                alt="linkedin icon"></a></li>
+          <li><a href="https://myaccount.google.com/?utm_source=sign_in_no_continue&pli=1"><img src="./Images/gmail.png"
+                alt="gmail icon"></a></li>
+          <li><a href="https://github.com/Khawanin"><img src="./Images/github.png" alt="github icon"></a></li>
+          <li><a href="https://www.google.com/"><img src="./Images/vector.png" alt="vector icon"></a></li>
+        </ul>
+      </section>
```

Comment:
- [ ] Kindly make sure the container of the section is a `<section>` tag for good semantics tag i.e you should wrap all the selected code in a `<section>` tag
- [ ] Kindly make sure you are using the correct **spacing**(margins or paddings) value at the `right` of the **Headline** content as the one used on your chosen Figma template.